src/pages/zh-cn/reference/configuration-reference.md

```diff
@@ -1,10 +1,14 @@
 ---
 layout: ~/layouts/MainLayout.astro
 title: 配置参考
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/%40types/astro.ts
 setup: |
   import Since from '../../../components/Since.astro';
+  import DontEditWarning from '../../../components/DontEditWarning.astro';
 ---
 
+<DontEditWarning />
+
```

Comment:
Ah, this warning component is specifically for the English page as that is auto-generated. We don’t need it for the translated version because we _do_ want people to update these manually!

```suggestion
```